frontend/internal-packages/artifact/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@liam-hq/artifact",
+  "description": "Artifact types and utilities for Liam ERD",
+  "keywords": [
+    "artifact",
+    "erd",
+    "types"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/liam-hq/liam",
+    "directory": "frontend/internal-packages/artifact"
+  },
+  "license": "Apache-2.0",
+  "version": "0.0.0",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./src/index.ts",
+  "exports": {
+    ".": {
+      "types": "./src/index.ts",
+      "default": "./src/index.ts"
+    }
+  },
+  "dependencies": {
+    "valibot": "1.1.0"
+  },
+  "devDependencies": {
+    "@biomejs/biome": "2.0.0",
+    "@liam-hq/configs": "workspace:*",
+    "@types/node": "22.15.30",
+    "typescript": "5.8.3"
+  },
+  "scripts": {
+    "dev": "tsc --watch",
+    "lint": "biome check .",
+    "lint:fix": "biome check --write .",
+    "lint:tsc": "tsc --noEmit"
+  },
```

Comment:
I want it to match the other packages. Do we need the dev command?

Reply:
You were right! I've pushed the fixes.https://github.com/liam-hq/liam/pull/2229/commits/236727b7fc86421e703aa912471358ab65292bd8